third_party/nvfuser/csrc/python_frontend/fusion_cache.h

```diff
@@ -99,9 +127,14 @@ class TORCH_CUDA_CU_API FusionCache {
   //! A pointer to the trie node in a cache lookup of a fusion definition.
   TrieNode* trie_ptr_;
   //! A vector of nvFuser Fusion IR fusions.
-  std::vector<std::unique_ptr<Nvf::FusionExecutorCache>> fusions_;
+  std::vector<FusionSchedules> fusions_;
   //! A vector of Terminal trie nodes for Stats collection
   std::vector<TrieNode*> terminal_nodes_;
+
+  // Inputs for user defined schedules are encoded into an integer Id
+  // NOTE: I would prefer this be per FusionSchedules object but the container
+  // is not allowed to be copied or moved.
+  Nvf::InputsIdLookup user_defined_input_encodings_;
```

Comment:
Is this used in this PR?

Reply:
It is not used, yet.  It will be used by user defined schedules.  The note tries to make clear the awkward placement in the abstraction.